### ISSUE TITLE
[Website Request] LibGroup (Manga) [RU]

### ISSUE THREAD

**Issue by NoPlagiarism** [MALSync/MALSync#2061]
### Website name

[MangaLib](https://mangalib.me), [SlashLib](https://slashlib.me)

### Website link

https://mangalib.me, https://slashlib.me

### Website language(s)

Russian

### Other details

It's big chain of websites that contains Russian translation for Manga. For simplicity "LibGroup" (Manga**Lib** and etc)

- One profile for all of websites, so it should be the same back-end for both of domains
- It has an API. 
    - For example. for ONIMAI ([AniList](https://anilist.co/manga/100080/ONIMAI-Im-Now-Your-Sister/), [MangaLib](https://mangalib.me/onii-chan-wa-oshimai)) it will be: `https://mangalib.me/manga-short-info?slug=onii-chan-wa-oshimai`
    - [Unofficial API Docs](https://github.com/NekonexDev/MangaLibAPI), [MangaLibAPI on Python](https://github.com/ArhostCode/MangaLibApi)

**Comment by NoPlagiarism** [comment]
Each overview and read page has `<script>` with `window.__DATA__ = ` with a lot of data.

Also, reading page has `window.__info = ` with data about current chapter, volume

**Comment by NoPlagiarism** [comment]
Temporarily closed until UI update.

For reference links to new UI:
[MangaLib](https://test-front.mangalib.me), [YaoiLib \(SlashLib\)](https://test-front.slashlib.me), [HentaiLib](https://test-front.hentailib.me)

**Comment by NoPlagiarism** [comment]
Reopening in case if anyone wants to take old UI or/and to escape any duplicates

**Comment by DFreezerR** [comment]
Do you think ranobe one should be implemented? I don't know if malsync have even a single light novel website support. But I think it might be possible since providers have light novels tracking. 

It just happened that I made some work on AnimeLib using their API, and since all of their websites uses the same UI + API, it will be much easier to implement others. Do you, by chance, have some work in progress regarding these websites? (Although I should've ask that before my PR...)

**Comment by henrik9999** [comment]
Btw Malsync already fully supports light novels, we just dont have any LN site currently implemented

**Comment by DFreezerR** [comment]
> Btw Malsync already fully supports light novels, we just dont have any LN site currently implemented

Good to know!

**Comment by NoPlagiarism** [comment]
> Do you, by chance, have some work in progress

Nope, I have no skills enough and too much perfectionism in my blood to deal with this SPA

So yeah, try. It would be nice if they will be implemented

**Comment by NoPlagiarism** [comment]
Fully closed with #2624 + #2631 + #2606

Thanks @DFreezerR for amazing work! 💜
